python/sglang/srt/models/mixtral.py

```diff
@@ -379,6 +382,25 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                         param, "weight_loader", default_weight_loader
                     )
                     weight_loader(param, loaded_weight)
+                    if self.torchao_config:
+                        if name.endswith("proj.weight") and param.ndim == 2:
+                            params_dict[name] = torchao_quantize_param_data(
+                                param, self.torchao_config
+                            )
+
+        if self.torchao_config:
```

Comment:
let us make a utility function for it to reduce the code duplication?

Reply:
yeah sure